core/src/main/scala/cats/data/XorT.scala

```diff
@@ -65,6 +65,9 @@ final case class XorT[F[_], A, B](value: F[A Xor B]) {
 
   def bimap[C, D](fa: A => C, fb: B => D)(implicit F: Functor[F]): XorT[F, C, D] = XorT(F.map(value)(_.bimap(fa, fb)))
 
+  def bitraverse[G[_], C, D](f: A => G[C], g: B => G[D])(implicit traverseF: Traverse[F], bitraverseXor: Bitraverse[Xor], applicativeG: Applicative[G]): G[XorT[F, C, D]] =
```

Comment:
Seems reasonable to me. I don't know why we have `traverseXorA` in the `XorT.traverse` signature. I would think we could remove that too.